MonoGame.Framework/Graphics/SpriteFont.cs

```diff
@@ -267,7 +267,7 @@ private void MeasureString(ref CharacterSource text, out Vector2 size)
                 }
                 hasCurrentGlyph = true;
 
-                var proposedWidth = offset.X + currentGlyph.WidthIncludingBearings + Spacing;
+                var proposedWidth = offset.X + currentGlyph.WidthIncludingBearings;

```

Comment:
We should be sure to include test cases of fonts with and without `Spacing` to be certain of the results.